roles/confluent.kafka_broker/tasks/main.yml

```diff
@@ -36,11 +45,11 @@
     keystore_storepass: "{{kafka_broker_keystore_storepass}}"
     keystore_keypass: "{{kafka_broker_keystore_keypass}}"
     service_name: kafka_broker
-    hostnames: "{{ kafka_broker_listeners | get_hostnames(inventory_hostname) | unique }}"
+    hostnames: "{{ [ansible_fqdn] + kafka_broker_listeners | get_hostnames(inventory_hostname) | unique }}"
```

Comment:
Might be worth adding a comment on how the listener string is built/where the values are pulled from.